content/docs/examples/creds-in-ocmconfig.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Credentials in ocmconfig file"
+description: ""
+summary: ""
+date: 2024-09-04T13:54:01+02:00
+lastmod: 2024-09-04T13:54:01+02:00
+draft: false
+weight: 999
+toc: true
+---
+
+## Overview
+
+The [OCM command line client](https://github.com/open-component-model/ocm/blob/main/docs/reference/ocm.md) can be configured by supplying it with a [configuration file](https://github.com/open-component-model/ocm/blob/main/docs/reference/ocm_configfile.md). By default, the CLI looks for configuration in `$HOME/.ocmconfig`, if it exists.
+
+The configuration file can be used in particular to specify the credentials, which are required for the CLI to be able to access the artifact repositories, referenced in CLI commands.
+
+## Examples
+
+This page contains basic examples of credentials configuration for a few most common artifact repository types. The examples below are complete `.ocmconfig` files, not snippets.
+
+For comprehensive documentation on the credentials topic, including usage of certificates or HashiCorp Vault, execute the command `ocm credential-handling`.
+
+### Repositories and Consumers
+
+In the examples below some configuration is located under `configurations[0]`.`repositories`, and some other under `configurations[0]`.`consumers`. This chanpter explains the difference between `repositories` and `consumers`, which is potentially not as obvious as one could think.

```

Comment:
```suggestion
In the examples below some configuration is located under `configurations.repositories`, and some other under `configurations.consumers`. This chanpter explains the difference between `repositories` and `consumers`, which is potentially not as obvious as one could think.

```

Although the notation `[0]` is technically correct, it can disturb the reading-flow. I would omit them

Reply:
Speaking of reading-flow, I'm one of the people, whose reading-flow would immediately be disturbed by incorrect technical references. My next questions would then be, what else might be wrong with the page, and if I can trust this technical documentation at all? Therefore I'd keep the index.